backend/src/database/migrations/20230608034909-create-govsg-tables.js

```diff
@@ -0,0 +1,300 @@
+'use strict'
+
+const { GovsgMessageStatus } = require('@core/constants')
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.query(
+      "ALTER TYPE enum_campaigns_type ADD VALUE 'GOVSG';"
+    )
+    await queryInterface.createTable('govsg_messages', {
+      id: {
+        autoIncrement: true,
+        type: Sequelize.DataTypes.BIGINT,
+        allowNull: false,
+        primaryKey: true,
+      },
+      campaign_id: {
+        type: Sequelize.DataTypes.BIGINT,
+        allowNull: true,
+        references: {
+          model: 'campaigns',
+          key: 'id',
+        },
+        onUpdate: 'CASCADE',
+      },
+      recipient: {
+        type: Sequelize.DataTypes.STRING(255),
+        allowNull: true,
+      },
+      params: {
+        type: Sequelize.DataTypes.JSONB,
+        allowNull: true,
+      },
+      service_provider_message_id: {
+        type: Sequelize.DataTypes.STRING(255),
+        allowNull: true,
+      },
+      error_code: {
+        type: Sequelize.DataTypes.STRING(255),
+        allowNull: true,
+      },
+      error_description: {
+        type: Sequelize.DataTypes.TEXT,
+        allowNull: true,
+      },
+      status: {
+        allowNull: true,
+        type: Sequelize.DataTypes.ENUM(Object.values(GovsgMessageStatus)),
+      },
+      accepted_at: {
+        type: Sequelize.DataTypes.DATE,
+        allowNull: true,
+      },
+      send_attempted_at: {
+        type: Sequelize.DataTypes.DATE,
+        allowNull: true,
+      },
+      sent_at: {
+        type: Sequelize.DataTypes.DATE,
+        allowNull: true,
+      },
+      delivered_at: {
+        type: Sequelize.DataTypes.DATE,
+        allowNull: true,
+      },
+      read_at: {
+        type: Sequelize.DataTypes.DATE,
+        allowNull: true,
+      },
+      errored_at: {
+        type: Sequelize.DataTypes.DATE,
+        allowNull: true,
+      },
+      created_at: {
+        type: Sequelize.DataTypes.DATE,
+        allowNull: false,
+      },
+      updated_at: {
+        type: Sequelize.DataTypes.DATE,
+        allowNull: false,
+      },
+    })
+    await queryInterface.createTable('govsg_templates', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      body: {
+        type: Sequelize.DataTypes.TEXT,
+        allowNull: true,
+      },
+      params: {
+        type: Sequelize.DataTypes.JSONB,
+        allowNull: true,
+      },
+      whatsapp_template_label: {
```

Comment:
maybe should be unique. see [discussion](https://opengovproducts.slack.com/archives/C05AQ9K7R2T/p1687503185358169?thread_ts=1687485697.075039&cid=C05AQ9K7R2T). 

Also see [this](https://github.com/opengovsg/postmangovsg/pull/2051/files#r1238600480)